build.ps1

```diff
@@ -50,26 +59,36 @@ function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishT
     Foreach-Object {
 
         # For self contained we want to process only IL ddls
+        <#
         if ($isSelfContained) {
             if (-Not [System.IO.File]::Exists($(Join-Path $regular $_.Name))) {
                 Write-Host "Skipping $_.Name"
                 Return
             }
         }
+        #>
 
         $prm = "/JITPath", "$publishTarget\download\clrjit\clrjit.dll", "/Platform_Assemblies_Paths", "$selfContained", "/nologo", "/in", $_.FullName
         # output for Microsoft.Azure.WebJobs.Script.WebHost.dll is Microsoft.Azure.WebJobs.Script.WebHost.exe.dll by default
         if ($_.FullName -like "*Microsoft.Azure.WebJobs.Script.WebHost.dll") {
             $prm += "/out"        
             $prm += Join-Path $privateSiteExtensionPath "Microsoft.Azure.WebJobs.Script.WebHost.ni.dll"
         }
+        # Fix output for System.Private.CoreLib.dll
+        if ($_.FullName -like "*System.Private.CoreLib.dll") {
+            $prm += "/out"        
+            $prm += Join-Path $privateSiteExtensionPath "System.Private.CoreLib.ni.dll"
+        }
 
-        & $crossGen $prm >> $buildOutput\crossgenout.$runtime.txt
+        & $crossGen $prm >> $buildOutput\crossgenout.$runtime.txt  2>&1
 
         $niDll = Join-Path $privateSiteExtensionPath $([io.path]::GetFileNameWithoutExtension($_.FullName) + ".ni.dll")
         if ([System.IO.File]::Exists($niDll)) {
             Remove-Item $_.FullName
             Rename-Item -Path $niDll -NewName $_.FullName
+
+            #& $crossGen "/Platform_Assemblies_Paths", "$selfContained", "/CreatePDB", "$symbolsPath", $_.FullName >> $buildOutput\crossgenout.$runtime.txt 2>&1
```

Comment:
Why is this commented out? wedo really need the pdbs

Reply:
Either way, let's remove commented out code. Either by re-adding or deleting.

Reply:
Yes, with CrossGen, we will need to recreate pdbs for .ni.dlls